pkg/sypgp/sypgp.go

```diff
@@ -840,6 +843,11 @@ func ExportPubKey(kpath string, armor bool) error {
 		return err
 	}
 
+	file, err := os.Create(kpath)
```

Comment:
Why don't you defer file.Close() right after this error check?

Reply:
The `defer file.Close()` is on line `854` or `862`. Although it makes sense to move it right under the error check. _I'll fix that..._